### PR TITLE
Adding procedure steps for all OS + examples

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

### CODE REVIEW

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 1]
```diff
@@ -5,11 +5,51 @@
 [id="using-{prod-id-short}-with-tls_{context}"]
 = Using {prod-short} with TLS
 
+Self-signed certificates are certificates that are not signed by a certificate authority (CA), and they do not provide all of the security properties that certificates signed by a CA aim to provide. For instance, when a website owner uses a self-signed certificate to provide HTTPS services, users who visit that website will see a warning in their browser.
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:8]: * , and they do not -> . Self-signed certificates do not
* For instance -> For example (not a mistake ofc, but it's simpler English)

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 2]
```diff
@@ -5,11 +5,51 @@
...
+
+After installing {prod-short} on {ocp} with TLS support and a self-signed certificate, a user has to add a self-signed TLS certificate to the browser for each route exposed in {prod-short}. If they do not, the login process will be blocked by the *Authorization token is missed. Click here to reload page* error message.
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:10]: I'd maybe rephrase this to highlight the problem we're trying to fix:
"...certificate, logging in to [what?] by default fails with an "Authorization token is missed. Click here to reload page" error message. To prevent this from occurring, add a self-signed certificate to the browser for each..."

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 3]
```diff
@@ -5,11 +5,51 @@
...
+
 This section describes how to import a root certificate authority into the browser to use {prod-short} with self-signed TLS certificates.
 
+== Adding Self-Signed Certificates to Chrome Browser using Linux and Windows OS
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:14]: * Self-Signed -> Self-signed
* If this is supposed to be in title case, then using -> Using
* to Chrome Browser -> to the Chrome Browser OR to Google Chrome
* using Linux and Windows OS -> on Linux or Windows

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 4]
```diff
@@ -5,11 +5,51 @@
...
+
+.Procedure
+
+. On the left side of the Chrome browser URL address bar panel, click the icon of the `Lock`, which represents the SSL/TLS status.
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:18]: * "of the Chrome browser URL address bar panel" is a bit of an over-classification, how about just "of Chrome's address bar"?
 * the icon of the `Lock` - the lock icon (if you're not gonna use an image, no sense in capsing and italicizing it.

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 5]
```diff
@@ -5,11 +5,51 @@
...
+
+. The information about your connection is displayed. Click *Certificates*.
+
+. Navigate to the *Details* tab.
+
+. Select the certificate you want to use and export it out using the btn:[Export] button.
++
+NOTE: Depending on the user’s OS, the *Export* (Linux-based OS) or *Save to File* (Windows-based OS) should be used.
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:26]: If I'm understanding this correctly, I'd suggest rephrasing:

...to use and export it:
+
* On a Linux OS, click btn:[Export].
* On a Windows-based OS, click btn:[Save to file]

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 6]
```diff
@@ -5,11 +5,51 @@
...
+
+.  On the right side of the browser click the *Customize and control Google Chrome* drop-down menu and navigate to *Settings*.
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:28]: browser click -> browser, click

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 7]
```diff
@@ -5,11 +5,51 @@
...
+
+. In the left panel, select *Advanced* and continue to *Privacy and security*.
+
+. In the middle of the screen, click *Manage certificates* and navigate to *Authorities* tab.
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:32]: In the middle of the screen -> At the center of the screen

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 8]
```diff
@@ -5,11 +5,51 @@
...
+
+. Add your certificate in the profile using the btn:[Import] button.
+
+
+== Adding Self-Signed Certificates to Chrome Browser using Mac OS
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:37]: * to Chrome Browser -> to the Chrome Browser OR to Google Chrome
* using Mac OS -> on Mac OS

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 9]
```diff
@@ -5,11 +5,51 @@
...
+
+.Procedure
+
+. On the left side of the Chrome browser URL address bar panel, click the icon of the `Lock`, which represents the SSL/TLS status.
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:41]: See comments on the similar line above

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 10]
```diff
@@ -5,11 +5,51 @@
...
+
+. The information about your connection is displayed. Click *Certificates*.
+
+. Select the certificate you want to use and then `drag and drop` its displayed bigger icon to the desktop.
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:45]: Not sure why drag and drop needs to be marked up :-)

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 11]
```diff
@@ -5,11 +5,51 @@
...
+
+. Double click on the exported certificate for importing into Chrome.
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:49]: * Double click on the -> Double-click the (!!!)
* for importing into Chrome -> to import it into Chrome (if I understand the meaning correctly)

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 12]
```diff
@@ -18,14 +58,13 @@ image::contributor/che-tls-chrome-import_1.png[link="{imagesdir}/contributor/che
 +
 image::contributor/che-tls-chrome-import_2.png[link="{imagesdir}/contributor/che-tls-chrome-import_2.png"]
 
-. Browser is satisfied with {prod-short} certificate and is reporting a secure connection.
+. After adding the {prod-short} certificate to the browser, the icon of `lock` is display next to the domain URL and browser reports a secure connection.
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:63]: See comments on similar lines above

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc [hunk 13]
```diff
@@ -38,13 +77,14 @@ image::contributor/che-tls-firefox-import_2.png[link="{imagesdir}/contributor/ch
 +
 image::contributor/che-tls-firefox-import_3.png[link="{imagesdir}/contributor/che-tls-firefox-import_3.png"]
 
-. Browser is satisfied with {prod-short} certificate and is reporting a secure connection.
+. After adding the {prod-short} certificate to the browser, the icon of `lock` is display next to the domain URL and browser reports a secure connection.
```
Comment (jherrman) [inline src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:82]: See comments on the similar line above

**Review by themr0c — APPROVED** [review]

File: src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc (hunk 9 above)
Comment (MichalMaler) [reply src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:41]: should be `address-bar panel` instead? :)

Comment (jherrman) [reply src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc:41]: Or maybe just "address bar", seems sufficiently descriptive to me.
